src/plugins/vis_augmenter/public/view_events_flyout/components/utils.tsx

```diff
@@ -47,9 +47,14 @@ function getValueAxisPositions(embeddable: VisualizeEmbeddable): { left: boolean
  * Fetching the base vis to show in the flyout, based on the saved object ID. Add constraints
  * such that it is static and won't auto-refresh within the flyout.
  * @param savedObjectId the saved object id of the base vis
+ * @param embeddableStart Optional EmbeddableStart passed in for plugins to utilize the function
  */
-export async function fetchVisEmbeddable(savedObjectId: string): Promise<VisualizeEmbeddable> {
-  const embeddableVisFactory = getEmbeddable().getEmbeddableFactory('visualization');
+export async function fetchVisEmbeddable(
+  savedObjectId: string,
+  embeddableStart?: EmbeddableStart
+): Promise<VisualizeEmbeddable> {
+  const embeddableLoader = embeddableStart !== undefined ? embeddableStart : getEmbeddable();
```

Comment:
Is this showing the same issues as UISettings etc. where it is not loaded when getting called by plugins?

Reply:
Yea I was having the same problem when calling it from the plugin (alerting/AD)